bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -971,18 +1155,79 @@ private void pasteSelectedPosition(final ISPObsComponent obsComponent, final Tar
         }
         @Override public void actionPerformed(final ActionEvent e) {
             // As long as something is selected, we can paste it.
-            _curSelection.forEach(c -> c.forEach(ignored -> pasteSelectedPosition(getNode(), getDataObject())));
-            _curSelection.forEach(c -> c.forEachLeft(ignored -> pasteSelectedPosition(getNode(), getDataObject())));
+            if (!(selection instanceof TableNoSelection))
+                pasteSelectedPosition(getNode(), getDataObject());
         }
     };
 
-    @SuppressWarnings("FieldCanBeLocal")
     private final ActionListener primaryListener = new ActionListener() {
         public void actionPerformed(final ActionEvent e) {
             _w.positionTable.updatePrimaryStar();
         }
     };
 
+    private final ActionListener linkedBaseListener = new ActionListener() {
+        private scala.Option<SPCoordinates> coords(final GhostAsterism ga, final boolean linked) {
+                if (linked)
+                    return ImOption.scalaNone();
+                return ImOption.toScalaOpt(
+                        ImOption.fromScalaOpt(ga.basePosition(ImOption.scalaNone()))
```

Comment:
Is `None` the thing to use for `when`?  Should it use the scheduling block instead?

Reply:
Yes, I was passing `None` for `when`. I didn't know we had scheduling block information available at that point, but if we do, I'll pass that in. (Not that it matters: I think it's ignored for sky positions anyway.)